### ISSUE TITLE
Remove warning about missing statement file

### ISSUE THREAD

**Issue by wil93** [olimpiadi-informatica/task-maker-rust#267]
When running `task-maker-rust` on a task, I get this warning:

```
Error: Missing statement file (statement/statement.pdf or testo/testo.pdf)
 Note: Without that file cms will not be able to import the task
 Help: Try: ln -s english.pdf testo/testo.pdf
```

This is not true anymore since https://github.com/cms-dev/cms/pull/1253 was merged, so it would make sense to either remove the warning, or to keep it but only show it if a `statement/<language_name>.pdf` file is missing (which would still be a useful warning).
